### PR TITLE
style: format command expansion server

### PR DESCRIPTION
**Pull request by camargo**
* **Tickets addressed:** None
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description

* Run Prettier format over command expansion server
* Change 'reformat' script to 'format'
* Alphabetize .prettierrc
* Regenerated test snapshot 
* No functional difference

## Verification

N/A

## Documentation

N/A

## Future work

N/A

### CODE REVIEW

**Comment by mattdailis** [comment]
Thoughts on starting a `.git-blame-ignore-revs` file? https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view

EDIT: Now that I think about it, we could do this retroactively, so we don't need to decide now

**Comment by camargo** [comment]
@mattdailis No preference. I don't use blame often personally. If you want to add this in a separate commit please go for it.

**Review by skovati — APPROVED** [review]
Looking quite pretty 👍

File: command-expansion-server/test/__snapshots__/command-types.spec.ts.snap [hunk 1]
```diff
@@ -299,20 +297,16 @@ export class Sequence {
...
-  return (
-\`export default () =>
+    return \`export default () =>
   Sequence.new({
     seqId: '\${this.seqId}',
     metadata: \${JSON.stringify(this.metadata)},
     commands: [\${commandsString}
     ],
-  });\`
-  );
```
Comment (Mythicaeda) [inline command-expansion-server/test/__snapshots__/command-types.spec.ts.snap:315]: Why was this return wrapped in `();`?

**Comment by mattdailis** [comment]
I'll add a commit, one sec :+1:

**Review by mattdailis — APPROVED** [review]
:+1: Thanks, I do use git blame quite heavily

**Review by Mythicaeda — APPROVED** [review]
LGTM!

File: command-expansion-server/test/__snapshots__/command-types.spec.ts.snap (hunk 1 above)
Comment (camargo) [reply command-expansion-server/test/__snapshots__/command-types.spec.ts.snap:315]: So this is the generated file but the [file that generates this code](https://github.com/NASA-AMMOS/aerie/pull/409/files#diff-224d12ae69810ebad6855e95127d0a73a31d2653d5207744b5f8f68370bc0727L304) had parens in the `return` statement. They are not needed tho so Prettier removed them.

**Pull request merged**